engine/gamesys/src/gamesys/resources/res_mesh.cpp

```diff
@@ -122,9 +122,7 @@ namespace dmGameSystem
         }
     }
 
-    bool BuildVertexDeclaration(BufferResource* buffer_resource,
-        dmGraphics::HVertexDeclaration* out_vert_decl,
-        uint32_t* out_elem_count, uint32_t* out_vert_size)
+    bool BuildVertexDeclaration(BufferResource* buffer_resource, dmGraphics::HVertexDeclaration* out_vert_decl)
```

Comment:
No need for the function to handle variables that can be retrieved from the buffer resource.